jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -265,7 +270,61 @@ Example: (send self :gripper :position) => 0.00"
             :touch-links nil)))
   (:delete-ground-collision-object ()
     (send *co* :delete-attached-object-by-id "ground")
-    (send *co* :delete-object-by-id "ground")))
+    (send *co* :delete-object-by-id "ground"))
+  (:servo-off (&key (arm t) (gripper t) (head t))
+    (let ((goal-servo-off (instance robot_controllers_msgs::QueryControllerStatesGoal :init))
+          (goal-gripper-servo-off (instance control_msgs::GripperCommandGoal :init))
+          (state nil) (update-list nil)
+          (gravity-comp (list "arm_controller/gravity_compensation"))
+          (arm-controller (list "arm_controller/follow_joint_trajectory"
+                                "arm_with_torso_controller/follow_joint_trajectory"
+                                "torso_controller/follow_joint_trajectory"))
+          (head-controller (list "head_controller/follow_joint_trajectory"
+                                 "head_controller/point_head")))
+      (when (or arm head)
+        (setq update-list (send goal-servo-off :updates))
+        (when arm
+          (ros::ros-info "arm servo off")
+          ;; start gravity compensation
+          (dolist (controller gravity-comp)
+            (setq state (instance robot_controllers_msgs::ControllerState :init))
+            (send state :name controller)
+            (send state :state 1) ;; running
+            (push state update-list))
+          ;; stop arm controllers
+          (dolist (controller arm-controller)
+            (setq state (instance robot_controllers_msgs::ControllerState :init))
+            (send state :name controller)
+            (send state :state 0) ;; stopping
+            (push state update-list)))
+        (when head
+          ;; stop head controllers
+          (ros::ros-info "head servo off")
+          (dolist (controller head-controller)
+            (setq state (instance robot_controllers_msgs::ControllerState :init))
+            (send state :name controller)
+            (send state :state 0) ;; stopping
+            (push state update-list)))
+        (send goal-servo-off :updates update-list)
+        (send fetch-controller-action :send-goal goal-servo-off))
+      (when gripper
+        ;; disable gripper torque
+        (ros::ros-info "gripper servo off")
+        (send (send goal-gripper-servo-off :command) :max_effort -1.0)
+        (send gripper-action :send-goal goal-gripper-servo-off))))
+  (:servo-on (&key (arm t) (gripper t) (head t))
+    ;; the reason why :servo-off and :servo-on has different implementation is in the following issue
+    ;; https://github.com/fetchrobotics/robot_controllers/issues/72
+    (when arm
+      (ros::ros-info "arm servo on")
+      (send self :angle-vector (send self :state :potentio-vector) 3000 :arm-controller)
```

Comment:
@tkmtnt7000 If 3000 is the minimum number to re-start controller, then it is ok. Otherwise, I we'd better to set this in keyword option.

Reply:
Thank you for reviewing. 
I confirmed minimum time to restart contollers. It is about 60 [msec] as follows.
So I'll set `:time` as keyword option. I'll set default time as 500 [msec] for now.
<details>
<summary>checking minimum time to restart controller </summary>

```
16.irteusgl$ send *ri* :servo-on :time 0
[ INFO] [1637381001.316778487] [/fetch_eus_interface_1637380539071598730]: [arm servo on]
[ INFO] [1637381001.525625578] [/fetch_eus_interface_1637380539071598730]: [;; Planned Trajectory Total Time   0.053 [sec]]
[ INFO] [1637381001.525788490] [/fetch_eus_interface_1637380539071598730]: [;; Scaled Trajectory Total Time 0.053(0.053) [sec]]
[ INFO] [1637381001.525845322] [/fetch_eus_interface_1637380539071598730]: [;; generated 2 points for 0.052928 sec using [arm_with_torso] group]
[ INFO] [1637381001.525893210] [/fetch_eus_interface_1637380539071598730]: [;; will send to (torso_lift_joint shoulder_pan_joint shoulder_lift_joint upperarm_roll_joint elbow_flex_joint forearm_roll_joint wrist_flex_joint wrist_roll_joint)]
[ INFO] [1637381001.525966171] [/fetch_eus_interface_1637380539071598730]: [;; send self :send-trajectory :arm-controller]
[ INFO] [1637381001.909813113] [/fetch_eus_interface_1637380539071598730]: [;; Planned Trajectory Total Time   0.064 [sec]]
[ INFO] [1637381001.909940000] [/fetch_eus_interface_1637380539071598730]: [;; Scaled Trajectory Total Time 0.064(0.064) [sec]]
[ INFO] [1637381001.909974174] [/fetch_eus_interface_1637380539071598730]: [;; generated 2 points for 0.063665 sec using [arm_with_torso] group]
[ INFO] [1637381001.910000743] [/fetch_eus_interface_1637380539071598730]: [;; will send to (torso_lift_joint shoulder_pan_joint shoulder_lift_joint upperarm_roll_joint elbow_flex_joint forearm_roll_joint wrist_flex_joint wrist_roll_joint)]
[ INFO] [1637381001.910035502] [/fetch_eus_interface_1637380539071598730]: [;; send self :send-trajectory :torso-controller]
[ INFO] [1637381001.913013224] [/fetch_eus_interface_1637380539071598730]: [head servo on]
[ INFO] [1637381002.203155503] [/fetch_eus_interface_1637380539071598730]: [;; Planned Trajectory Total Time   0.066 [sec]]
[ INFO] [1637381002.203274844] [/fetch_eus_interface_1637380539071598730]: [;; Scaled Trajectory Total Time 0.066(0.066) [sec]]
[ INFO] [1637381002.203311426] [/fetch_eus_interface_1637380539071598730]: [;; generated 2 points for 0.065561 sec using [arm_with_torso] group]
[ INFO] [1637381002.203357681] [/fetch_eus_interface_1637380539071598730]: [;; will send to (torso_lift_joint shoulder_pan_joint shoulder_lift_joint upperarm_roll_joint elbow_flex_joint forearm_roll_joint wrist_flex_joint wrist_roll_joint)]
[ INFO] [1637381002.203419477] [/fetch_eus_interface_1637380539071598730]: [;; send self :angle-vector :head-controller (#<controller-action-client #X563271925ec0 /head_controller/follow_joint_trajectory>) (without planning)]
[ INFO] [1637381002.210564802] [/fetch_eus_interface_1637380539071598730]: [;; send self :send-trajectory :head-controller]
[ INFO] [1637381002.210622252] [/fetch_eus_interface_1637380539071598730]: [gripper servo on]
#<control_msgs::grippercommandresult #X56327fd17318>
```

</details>